app/services/attribute_asserter.rb

```diff
@@ -200,7 +200,10 @@ def attribute_getter_function_ascii(attr)
 
   def add_email(attrs)
     attrs[:email] = {
-      getter: ->(principal) { EmailContext.new(principal).last_sign_in_email_address.email },
+      getter: ->(principal) {
```

Comment:
Need test coverage for this in `spec/services/attribute_asserter_spec.rb`

Reply:
Added test for that